### PR TITLE
Add oneNotePickerDropdown to oneNotePicker

### PR DESCRIPTION
**Pull request by JaneHazen**
Add oneNotePickerDropdown to oneNotePicker

### CODE REVIEW

File: src/oneNotePicker.tsx [hunk 1]
```diff
@@ -64,5 +64,5 @@ export class OneNotePicker extends OneNotePickerBase<OneNotePickerProps, {}> {
 }
 
 export * from './oneNoteSingleNotebookPicker';
-export * from './oneNoteSingleNotebookDropdown';
+export * from './oneNotePickerDropdown';
```
Comment (jogonzal) [inline src/oneNotePicker.tsx:67]: Did you mean to remove oneNoteSingleNotebookDropdown?

**Review by jogonzal — APPROVED** [review]

File: src/oneNotePicker.tsx (hunk 1 above)
Comment (JaneHazen) [reply src/oneNotePicker.tsx:67]: Yes- it is in the oneNotePickerDropdown file

Comment (mttwc) [reply src/oneNotePicker.tsx:67]: I think we should still keep it in (more resilient against refactor). We should explicitly keep all the widgets as exports in this file.

**Review by mttwc — CHANGES_REQUESTED** [review]

**Review by mttwc — APPROVED** [review]

File: src/oneNotePicker.tsx (hunk 1 above)
Comment (mttwc) [reply src/oneNotePicker.tsx:67]: As discussed, you're right in doing this :)

**Pull request merged**